### PR TITLE
docs: add ec2 BR

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

## Summary by CodeRabbit

- **Documentation**
  - Added a link to Amazon EC2's Backup and Recovery solution in the English user guide for disaster recovery scenarios.
  - Updated the Chinese user guide with additional information on deploying GreptimeDB Standalone using backup and recovery solutions on IaaS platforms like Amazon EC2 with EBS volumes.

### CODE REVIEW

File: docs/nightly/en/user-guide/operations/disaster-recovery/overview.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ If the Standalone is running on the local disk for WAL and data, then:
 * RPO: depends on backup frequency.
 * RTO: doesn't make sense in standalone mode, mostly depends on the size of the data to be restored, your failure response time, and the operational infrastructure.
 
+Amazon EC2 with EBS volumes provides a comprehensive [Backup and Recovery solution](https://docs.aws.amazon.com/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html) for such scenarios, and the other IaaS vendors also offer similar solutions. Choosing this one is a good start.
```
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:61]: > Choosing this one is a good start.

which one?

Comment (nicecui) [reply docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:61]: We are introducing GreptimeDB's solution, but here we introduce the solution of AWS. The logic isn't smooth.

Comment (killme2008) [reply docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:61]: Please read the context.

File: docs/nightly/zh/user-guide/operations/disaster-recovery/overview.md [hunk 2]
```diff
@@ -66,6 +66,8 @@ GreptimeDB 将数据存储在对象存储（如 [AWS S3](https://docs.aws.amazon
 * RPO：取决于备份频率。
 * RTO：在 Standalone 没有意义，主要取决于要恢复的数据大小、故障响应时间和操作基础设施。
 
+亚马逊EC2（结合 EBS卷）提供了一个全面的[备份和恢复解决方案](https://docs.aws.amazon.com/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html)来应对这种情况，当然其他IaaS供应商也提供类似的解决方案。选择这个方案是一个很好的开始。
```
Comment (nicecui) [inline docs/nightly/zh/user-guide/operations/disaster-recovery/overview.md:69]:
```suggestion
选择将 GreptimeDB Standalone 部署到具有备份和恢复解决方案的 IaaS 平台中是一个很好的开始，例如亚马逊 EC2（结合 EBS卷）提供了全面的[备份和恢复解决方案](https://docs.aws.amazon.com/zh_cn/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html)。
```

File: docs/nightly/en/user-guide/operations/disaster-recovery/overview.md (hunk 1 above)
Comment (nicecui) [inline docs/nightly/en/user-guide/operations/disaster-recovery/overview.md:61]:
```suggestion
A good start is to deploy GreptimeDB Standalone into an IaaS platform that has a backup and recovery solution. For example, Amazon EC2 with EBS volumes provides a comprehensive [Backup and Recovery solution](https://docs.aws.amazon.com/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html).
```

**Review by nicecui — APPROVED** [review]

File: docs/nightly/zh/user-guide/operations/disaster-recovery/overview.md [hunk 3]
```diff
@@ -66,6 +66,8 @@ GreptimeDB 将数据存储在对象存储（如 [AWS S3](https://docs.aws.amazon
 * RPO：取决于备份频率。
 * RTO：在 Standalone 没有意义，主要取决于要恢复的数据大小、故障响应时间和操作基础设施。
 
+选择将 GreptimeDB Standalone 部署到具有备份和恢复解决方案的 IaaS 平台中是一个很好的开始，例如亚马逊 EC2（结合 EBS卷）提供了全面的[备份和恢复解决方案](https://docs.aws.amazon.com/zh_cn/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html)。
```
Comment (fengjiachun) [inline docs/nightly/zh/user-guide/operations/disaster-recovery/overview.md:69]:
```suggestion
选择将 GreptimeDB Standalone 部署到具有备份和恢复解决方案的 IaaS 平台中是一个很好的开始，例如亚马逊 EC2（结合 EBS 卷）提供了全面的[备份和恢复解决方案](https://docs.aws.amazon.com/zh_cn/prescriptive-guidance/latest/backup-recovery/backup-recovery-ec2-ebs.html)。
```

**Review by fengjiachun — APPROVED** [review]

**Pull request merged**